taichi/util/bit.h

```diff
@@ -121,6 +121,10 @@ TI_FORCE_INLINE constexpr uint32 log2int(uint64 value) {
   return ret;
 }
 
+TI_FORCE_INLINE constexpr uint32 ceil_log2int(uint64 value) {
+  return log2int(value) + ((value & (value - 1)) != 0);
```

Comment:
```suggestion
  return log2int(value) + 1 - (int)is_power_of_two(value);
```
(what if `value=0`?)

Reply:
When `value==0`, `log2int(value) + ((value & (value - 1)) != 0)` gives `0`, but `log2int(value) + 1 - (int)is_power_of_two(value)` gives `1` because 0 is not a power of 2.

Reply:
Yeah, I mean, should we document this behavior to the function? Or we should simply forbid passing in `0`?

Reply:
Oh, I think we should document it.